ow_faults/msg/SystemFaults.msg

```diff
@@ -9,6 +9,7 @@
 # PtGoalError=64, 
 # PtExecutionError=128, 
 # LanderExecutionError = 256
```

Comment:
I would expect to see a comma at the end of this line. Either maintain the commas or remove them

Reply:
Note this is the line before last